examples/014_cyclic_symmetry_example.py

```diff
@@ -0,0 +1,160 @@
+# Copyright (C) 2023 - 2024 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+"""
+.. _cyclic_symmetry_example:
+
+Cyclic symmetry
+---------------
+
+This example shows how to post-process a cyclic symmetry analysis.
+The initial (original) sector can be post-processed with the same tools
+as a standard analysis. This is demonstrated by running a failure analysis,
+extracting ply-wise stresses and the implementation of a custom
```

Comment:
Question about this sentence (including the one revised below): Is the failure analysis extracting both ply-wise stresses and the implementation, or extracting only the ply-wise stresses along with implementing a custom failure criterion?

```suggestion
 The postprocessing workflow is demonstrated by running a failure analysis that extracts ply-wise stresses and the implementation of a custom failure criterion.
```

Reply:
Hi @nshum4 , these are 3 independent examples: 
- failure analysis
- extracting ply-wise stresses
- implementation of a custom failure criterion

I agree, it's not clear how it is written currently.